### PR TITLE
Fix minor bug when orientation changes for mode sectionHorizontalContentVertical

### PR DESCRIPTION
**Pull request by PanajotisMaroungas**

### CODE REVIEW

**Comment by hebertialmeida** [comment]
Is it to fix #111? I tested but not fixed that.

**Comment by PanajotisMaroungas** [comment]
No, it fixes problem for the swipe/scroll mode. The Font menu bug is in every scrollDirection. 
I tested the latest pods (which doesn’t include the last PR) and it bug existed.
I think it will be fixed with some constraints. Should I check it?

**Comment by hebertialmeida** [comment]
We have to do a lot of hacks because this new layout direction, because when I created this generic function I did not prevent your new layout.

I think the best and less buggy what is to refactor this func:
https://github.com/FolioReader/FolioReaderKit/blob/master/Source/FolioReaderKit.swift#L210

To support 3 layouts, so we don't need if else for this.

```
func isDirection<T> (vertical: T, _ horizontal: T, _ horizontalContentVertical: T) -> T {
    switch readerConfig.scrollDirection {
    case .vertical: return vertical
    case .horizontal: return horizontal
    case .sectionHorizontalContentVertical: return horizontalContentVertical
    }
}
```

Also I think we can rename the enum case `sectionHorizontalContentVertical` to `horizontalWithVerticalContent`, what so you think?

**Comment by PanajotisMaroungas** [comment]
You’re right. I will start with this now!

File: Source/FolioReaderCenter.swift [hunk 1]
```diff
@@ -1020,21 +1017,15 @@ extension FolioReaderCenter: FolioReaderPageDelegate {
...
             let pageNumber = position["pageNumber"]! as! Int
-            var pageOffset: CGFloat = 0
-
-			if readerConfig.scrollDirection == .sectionHorizontalContentVertical,
-				let offSetY = position["pageOffsetY"] as? CGFloat {
-					pageOffset = offSetY
-			} else if let offset = isVerticalDirection(position["pageOffsetY"], position["pageOffsetX"]) as? CGFloat {
-				pageOffset = offset
-			}
+			let offset = isDirection(position["pageOffsetY"], position["pageOffsetX"], position["pageOffsetY"]) as? CGFloat
```
Comment (hebertialmeida) [inline Source/FolioReaderCenter.swift:1020]: Saved many lines of code 💥

**Pull request merged**